doc/source/whatsnew/v1.5.0.rst

```diff
@@ -796,6 +796,8 @@ Reshaping
 - Bug in :func:`concat` with identical key leads to error when indexing :class:`MultiIndex` (:issue:`46519`)
 - Bug in :meth:`DataFrame.join` with a list when using suffixes to join DataFrames with duplicate column names (:issue:`46396`)
 - Bug in :meth:`DataFrame.pivot_table` with ``sort=False`` results in sorted index (:issue:`17041`)
+- Bug in :func:`merge` and :meth:`DataFrame.merge` now allows passing ``None`` or ``(None, None)`` for ``suffixes`` argument, keeping column labels unchanged in the resulting :class:`DataFrame` potentially with duplicate column labels (:issue:`46885`)
```

Comment:
let's move these to other enhancements section (change the text as these are not bugs per se, rather a change in api)

Reply:
you can reference the original issue as a bug report if you want (just make it a single note). but i *also* want a note in enhancements.

Reply:
moved to enhancements